package.json

```diff
@@ -2,6 +2,10 @@
   "name": "etcher",
   "displayName": "Etcher",
   "version": "1.0.0-rc.4",
+  "updates": {
+    "semverRange": "<2.0.0",
+    "useUnstableChannel": true
```

Comment:
I can't understand how you manage to catch every single little thing. I'm jealous :P